app/models/user.py

```
@@ -39,7 +39,6 @@ class User(JSONModel, UserMixin):
         'mobile_number',
         'password_changed_at',
         'permissions',
-        'platform_admin',
```

How come we get rid of this property?

---

Properties shouldn’t be in this list if they’re [already implemented on the class](https://github.com/alphagov/notifications-admin/blob/92b735f6e471473c85372b5e4fa4e0949a806ccb/app/models/user.py#L190-L192). Trying to override a property that’s already implemented on the class using `setattr` will raise an exception.

---

does the `not hasattr(self, property)` make sure it doesn't raise the exception?

Although either way it's fine to remove this line as you are right that it's already implemented.

---

I’ve updated it so it now raises an exception where there’s this conflict, rather than resolving silently. This will help us keep the `ALLOWED_PROPERTIES` list tidy. Changes in 38e9e84f7